pkg/actions/addon/addon.go

```diff
@@ -86,6 +87,14 @@ func (a *Manager) waitForAddonToBeActive(addon *api.Addon) error {
 }
 
 func (a *Manager) getLatestVersion(addon *api.Addon) (string, error) {
```

Comment:
do we wanna rename this now? Its sort of "get the latest version matching my string" 😂 , maybe "getLatestMatchingVersion" or something to the effect, wdyt?